Lib/ruby/rubycontainer.swg

```diff
@@ -101,7 +101,8 @@ namespace swig {
   inline Sequence*
   getslice(const Sequence* self, Difference i, Difference j) {
     typename Sequence::size_type size = self->size();
-    typename Sequence::size_type ii = swig::check_index(i, size, (i == size && j == size));
+    typename Sequence::size_type ii = swig::check_index(i, size,
+      (i == (Difference)size && j == (Difference)size));
```

Comment:
Why do you put this on two lines, please put it back. Same for getslice.

Reply:
Done